improve the migration guide for configuring the start and end convers… (#3346)
BetonQuest/BetonQuest: docs/Documentation/Configuration/Version-Changes/Migration-2-3.md

</div>

If you want to disable the the start and end message, now you can configure the following:

````YAML title="Disable Messages"
notifications:
  conversation_start:
    io: suppress
  conversation_end:
    io: suppress
````

> and if you only want sounds and no message, you configure the io to sound like this: `io: sound`.

Reviewer: ```suggestion
and if you only want sounds and no message, you use `sound` instead of `suppress` as io.
```
